python/gherkin3/token_matcher.py

```diff
@@ -50,13 +91,11 @@ def match_StepLine(self, token):
             title = token.line.get_rest_trimmed(len(keyword))
             self._set_token_matched(token, 'StepLine', title, keyword)
             return True
-
         return False
 
     def match_Comment(self, token):
         if not token.line.startswith('#'):
             return False
-
```

Comment:
> No, it not necessary to have a blank line separating the guard clause from the main work of the method, but do you think it is so bad, that it motivates going through the whole method and remove them? 

No, it doesn’t.  I did not intend any negative statement about those spaces.  Sorry.  :(

> I try hard not to ask because you propose changes in code I originally wrote, but from the perspective I try to have when evolving existing code - "be consistent with what is already there" (unless there is a very good reason).

I understand this mindset.  I will do my best to respect it from now on.  

I see that many of my changes are not to your liking, or @aslakhellesoy ’s.  I love this project, and I’d like to contribute.  But it’s frustrating to discover that my PRs seem violate a lot of undocumented expectations of the code.  

Have I wasted everyone’s time with this PR?  I don’t know.  Maybe…maybe not.  But I think it would be a good idea to document things like your PEP8 `--max-line-length`, and the “be consistent with what is already there (unless there is a very good reason)” mentality to review.  There is nothing wrong with that mentality, but there _are_ projects who are perfectly okay with the sorts of changes in this PR. Others share the mentality, but have different ideas about what is a “good reason”.  Where is a contributor supposed to learn where you stand on this?